3rdparty/opus-build/opus-build.pro

```diff
@@ -20,6 +20,11 @@ VPATH = ../$$SOURCEDIR
 TARGET = opus
 DEFINES += HAVE_CONFIG_H
 
+CONFIG(static) {
```

Comment:
Doesn't this also change behaviour for windows? How did we get the right binary there?

Reply:
The library was already forced to be compiled as shared on Windows:
https://github.com/mumble-voip/mumble/blob/15072a4505af39f254fe0d35357d95b9a121e991/3rdparty/opus-build/opus-build.pro#L36-L37

Reply:
I removed those lines, since they are redundant.

Reply:
That means we also introduce it for Linux though!?

Reply:
That's correct.